File: lib/models/response.dart
class Response {
  String id;
  String questionId;
  String userId;
  String userName;
  String body;
  String createdDate;
  int upvotes;

  Response(
      {required this.id,
      required this.questionId,
      required this.userId,
      required this.userName,
      required this.body,
      required this.createdDate,
      this.upvotes = 1});

  factory Response.fromJson(Map<String, dynamic> json) {
    return Response(
      id: json['id'] as String,
      questionId: json['questionId'] as String,
      userId: json['userId'] as String,
      userName: "Loading...",
      body: json['body'] as String,
      createdDate: json['date'] as String,
      upvotes: json['upvotes'] as int,
    );
  }
}

File: backend/Models/Response.cs
namespace ecommerceapp.models;
public class Response {
    public string Id { get; set; }
    public string User { get; set; }
    public string Body { get; set; }
    public string createdDate { get; set; }
    public int Upvotes;

    public Response(string Id, string User, string Body, string createdDate, int Upvotes) {
        this.Id = Id;
        this.User = User;
        this.Body = Body;
        this.createdDate = createdDate;
        this.Upvotes = Upvotes;
    }
}
